### PR TITLE
Jobs backend

### PR DESCRIPTION
**Pull request by arikalon1**

### CODE REVIEW

File: src/robusta/core/model/jobs.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    created_at: str
+    deleted: bool = False
+    cpu_req: float
+    mem_req: int
+    completions: int
+    status: JobStatus
+    job_data: JobData
+
+    def get_service_key(self) -> str:
+        return f"{self.namespace}/{SERVICE_TYPE_JOB}/{self.name}"
```
Comment (Sheeproid) [inline src/robusta/core/model/jobs.py:96]: /{type}/ ?

File: src/robusta/core/model/jobs.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+    def __eq__(self, other):
+        if not isinstance(other, JobInfo):
+            return NotImplemented
+
+        return (  # ignore created_at because of dates format
```
Comment (Sheeproid) [inline src/robusta/core/model/jobs.py:102]: why should date formatting be different in the same robusta instance?  or it is because of db row?

**Review by Sheeproid — APPROVED** [review]

File: src/robusta/core/sinks/robusta/dal/supabase_dal.py [hunk 3]
```diff
@@ -133,16 +134,19 @@ def to_service(self, service: ServiceInfo) -> Dict[Any, Any]:
...
-            self.client.table(SERVICES_TABLE).insert(db_service, upsert=True).execute()
+            self.client.table(SERVICES_TABLE).insert(db_services, upsert=True).execute()
         )
         if res.get("status_code") not in [200, 201]:
             logging.error(
-                f"Failed to persist service {service} error: {res.get('data')}"
+                f"Failed to persist services {services} error: {res.get('data')}"
             )
             self.handle_supabase_error()
+            raise Exception("publish service failed")
```
[the remark this answers is not in the document]
Comment (arikalon1) [reply src/robusta/core/sinks/robusta/dal/supabase_dal.py:149]: fixed

File: src/robusta/core/model/jobs.py (hunk 1 above)
Comment (arikalon1) [reply src/robusta/core/model/jobs.py:96]: fixed

**Pull request merged**